assets/js/updaters.js

```diff
@@ -269,11 +269,11 @@ function updateVoidTraderInventory() {
       /* eslint-disable prefer-template */
       const inventoryString = `${'<div class="panel panel-primary voidTraderInventory" ' +
                 'style="margin-left:5%; margin-right:5%" ' +
-                'id="'}${worldState.voidTrader.id}">\n<div class="panel-heading">\n` +
+                'id="'}${worldState.voidTrader.id}">\n` +
+                '<a href="#voidTraderInventoryPanel" data-toggle="collapse" class="panel-heading show">\n' +
```

Comment:
could you add a text-center to the anchor classes here, that way it's consistent with the other panels?